### ISSUE TITLE
Odo catalog list and search shows hidden services and components

### ISSUE THREAD

**Issue by girishramnani** [redhat-developer/odo#1180]
[kind/bug]

## What versions of software are you using?
- Operating System:
- Output of `odo version`:
```
odo v0.0.17 (eec9a26e)

Server: https://master.odof2f-5eeb.openshiftworkshop.com:443
OpenShift: v3.11.43
Kubernetes: v1.11.0+d4cacc0
```

## How did you run odo exactly?
`odo catalog list components`
`odo catalog list services`

## Actual behavior
Shows all components and services

## Expected behavior
Shouldn't show services and components that are "hidden"

**Comment by geoand** [comment]
@girishramnani if you need any help with this ( which you probably won't :) ), don't hesitate to ask

**Comment by girishramnani** [comment]
this PR resolves services part https://github.com/redhat-developer/odo/pull/1181

**Comment by amitkrout** [comment]
Resolved via #1181

**Comment by girishramnani** [comment]
Actually @amitkrout only the services part of this issue was resolved by #1181, hidden components are still showing

**Comment by amitkrout** [comment]
My bad..
Thanks for reopening it.
